swing/src/main/java/info/openrocket/swing/gui/configdialog/RocketComponentConfig.java

```diff
@@ -462,6 +462,10 @@ public void selectPreset() {
 			@Override
 			public void run() {
 				if (presetComboBox == null || presetModel == null) return;
+				if (preferences.getBoolean(component.getComponentName() + "AlwaysOpenPreset", preferences.getAutoOpenPartsLibrary()) != preferences.getAutoOpenPartsLibrary()) {
```

Comment:
Please remove this, because this would mean that you would not be able to open the parts dialog _at all_ anymore.